recipes/libxml2/all/conanfile.py

```diff
@@ -255,9 +257,13 @@ def package_info(self):
             self.env_info.PATH.append(bindir)
 
         if self.settings.os == "Linux" or self.settings.os == "Macos":
-            self.cpp_info.system_libs.append('m')
+            self.cpp_info.system_libs.append("m")
         if self.settings.os == "Windows":
-            self.cpp_info.system_libs.append('ws2_32')
-        self.cpp_info.names["cmake_find_package"] = "LibXml2"
-        self.cpp_info.names["cmake_find_package_multi"] = "LibXml2"
+            self.cpp_info.system_libs.append("ws2_32")
+        if self.options.threads:
+            if self.settings.os == "Linux":
+                self.cpp_info.system_libs.append("pthread")
+
         self.cpp_info.names["pkg_config"] = "libxml-2.0"
+        self.cpp_info.filenames["cmake_find_package"] = "libxml2"
```

Comment:
https://cmake.org/cmake/help/v3.18/module/FindLibXml2.html

Reply:
Ah oke. I did only look at the libxml2-config.cmake file created by the package itself.